frontend/model/contracts/group.js

```diff
@@ -50,14 +54,15 @@ export function createInvite (
   responses: {...},
   status: string,
 |} {
+  const expiresInDate = creator === INVITE_INITIAL_CREATOR ? INVITE_EXPIRES_IN_DAYS.INITIAL : INVITE_EXPIRES_IN_DAYS.PROPOSAL
```

Comment:
This PR is looking really good! One request that I have is to store the expiry time values in the group settings so that in the future if we need to change them (or allow the group to change them), we can.

Here's how I recommend doing that:

Modify the group constructor to include these values:

```js
      process ({ data, meta }, { state, getters }) {
        // TODO: checkpointing: https://github.com/okTurtles/group-income-simple/issues/354
        const initialState = merge({
          payments: {},
          paymentsByMonth: {},
          distributionEvents: [],
          invites: {},
          proposals: {}, // hashes => {} TODO: this, see related TODOs in GroupProposal
          settings: {
            groupCreator: meta.username,
            inviteExpiryInitial: INVITE_EXPIRES_IN_DAYS.INITIAL,
            inviteExpiryProposal: INVITE_EXPIRES_IN_DAYS.PROPOSAL
          },
```

Then, modify this function to take the expiry time as a variable:

```js
export function createInvite (
  {
    quantity = 1,
    creator,
    expires,
    invitee
  }: { quantity: number, creator: string, expires: number, invitee?: string }
): {|
```

And when it is called in the UI, pass in the expiry for the proposal based on the current value:

```js
        if (oneVoteToPass(proposalHash)) {
          if (this.type === PROPOSAL_INVITE_MEMBER) {
            passPayload = createInvite({
              invitee: this.proposal.data.proposalData.member,
              creator: this.proposal.meta.username,
              expires: this.currentGroupState.settings.inviteExpiryProposal
            })
```

Reply:
Your recommendation is actually nice. But you didn't mention how to set pass expires when we create the initial invite.
We call this sbp function inside `frontned/controller/actions/group.js` when we create the group itself.
So you mean, we can only manage the expiresInDate for the proposal-based invitations.
In other words, we don't use the variable defined as `group.settings.inviteExpiryInitial`.